src/main/scala/org/globalforestwatch/util/IntersectGeometry.scala

```diff
@@ -0,0 +1,69 @@
+package org.globalforestwatch.util
+
+import geotrellis.vector.{Geometry, Polygon, Point}
+import org.locationtech.jts.geom.Coordinate
+
+object IntersectGeometry {
+
+  def getIntersecting1x1Grid(geometry: Geometry): IndexedSeq[Polygon] = {
+
+    /**
+      * Return grid of 1x1 degree polygons intersecting with input geometry
+      *   */
+    val coords: Array[Coordinate] = geometry.getEnvelope.getCoordinates
+    val (minX, minY, maxX, maxY) = coords.foldLeft(180.0, 90.0, -180.0, -90.0)(
+      (z, coord) =>
+        (coord.x min z._1, coord.y min z._2, coord.x max z._3, coord.y max z._4)
+    )
+
+    for {
+      x <- minX.floor.toInt until maxX.ceil.toInt
+      y <- minY.floor.toInt until maxY.ceil.toInt
+    } yield {
+      val polygon: Polygon = Polygon(
```

Comment:
I guess we haven't really tested this yet, but do you think it's possible the RADD/GLAD+ resolution tiles are going to require smaller tiles to work efficiently?